services/github/github.go

```diff
@@ -794,11 +794,15 @@ func (s *Service) newIssueLabels(roomID id.RoomID) []string {
 	if err != nil {
 		return make([]string, 0)
 	}
-	newIssueLabels, ok := ghOpts["new_issue_labels"].(string)
+	newIssueLabels, ok := ghOpts["new_issue_labels"].([]interface{})
 	if !ok {
 		return make([]string, 0)
 	}
-	return strings.Split(newIssueLabels, ",")
+	newIssueLabelsUnboxed := make([]string, 0)
```

Comment:
I don't actually know go, so if someone knows how to perform this conversion without doing a loop, please shout

Reply:
This is how you do it in Go, sadly. At least I haven't found a better way yet...

Reply:
Unmarshaling the options into a struct with the proper types instead of a map might be a better way, but that would also be a more complicated change